src/test/java/HWTestForLoginTest.java

```diff
@@ -0,0 +1,66 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+    public class HWTestForLoginTest {
+    @Test
+    void test1 () throws InterruptedException {
+        //1. Login Success.
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://kwidos.com/");
+        Thread.sleep(2000);
+        driver.findElement(By.xpath("//*[@class='glyphicon glyphicon-log-in']/../p")).click();
+        Thread.sleep(2000);
+        driver.findElement(By.id("email")).sendKeys("alekseimaslo@gmail.com");
+        driver.findElement(By.id("password")).sendKeys("Kwidos123!");
+        driver.findElement(By.xpath("//*[@type='submit']")).click();
+
+        driver.quit();
+    }
+
+    @Test
+    void test2 () throws InterruptedException {
+        //2. Login Wrong credentials (not existing email)
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://kwidos.com/");
+        Thread.sleep(2000);
+        driver.findElement(By.xpath("//*[@class='glyphicon glyphicon-log-in']/../p")).click();
+        Thread.sleep(2000);
+        driver.findElement(By.id("email")).sendKeys("alekseimaslo@gmail.com");
+        driver.findElement(By.id("password")).sendKeys("Kwidos123!");
+        driver.findElement(By.xpath("//*[@type='submit']")).click();
+        Thread.sleep(2000);
+        WebElement alert = driver.findElement(By.xpath("//div[@class='alert alert-danger']"));
+        Assert.assertTrue(alert.isDisplayed());
+
+        driver.quit();
+    }
+
+    @Test
+        void test3 () throws InterruptedException {
+        //3. Login Empty credentials
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://kwidos.com/");
+        Thread.sleep(2000);
+        driver.findElement(By.xpath("//*[@class='glyphicon glyphicon-log-in']/../p")).click();
+        Thread.sleep(2000);
+
+        driver.findElement(By.id("email")).sendKeys("");
+        driver.findElement(By.id("password")).sendKeys("");
+        driver.findElement(By.xpath("//*[@type='submit']")).click();
```

Comment:
You use the same locators for different tests. You can store locators in separate static final variables and use one locator for all tests. It is a good practice because if locator will change you can fix it in one place and all will work